### PR TITLE
[webui] Refactoring project new

### PR DESCRIPTION
**Pull request by mdeniz**

### CODE REVIEW

File: src/api/app/views/webui/project/new.html.erb [hunk 1]
```diff
@@ -1,52 +1,15 @@
...
 %>
 <% project_bread_crumb 'New Project' %>
 
 <h3><%= @pagetitle %></h3>
 <% unless User.current.is_admin? %>
   <p>
-    All users have a <b>home:&lt;login&gt;</b> project by default, and can also create new sub projects. New project outside of your home namespace can get requested via
-    <a href="https://bugzilla.opensuse.org/enter_bug.cgi?classification=7340&amp;product=openSUSE.org&amp;submit=Use+This+Product&amp;component=OBS%20Request">bugzilla</a>.
-    To discuss the requested project setup, please use the <a href="mailto:opensuse-buildservice@opensuse.org">opensuse-buildservice@opensuse.org</a> mailing list.
+    All users have a homes project by default. Yours is <%= link_to "home:#{User.current.login}", { controller: :project, action: :show,
```
Comment (hennevogel) [inline src/api/app/views/webui/project/new.html.erb]: homes -> home

File: src/api/app/views/webui/project/new.html.erb [hunk 2]
```diff
@@ -1,52 +1,15 @@
...
+    project: "home:#{User.current.login}" } %>, and you can also
+    create sub projects inside. New projects outside of your home project can get
+    requested via <a href="mailto:<%= @configuration['admin_email'] %>">email</a>.
```
Comment (hennevogel) [inline src/api/app/views/webui/project/new.html.erb]: > All users have a homes project by default. Yours is <%= link_to %>, and you can also
> create sub projects inside. New projects outside of your home project can get
> requested via &lt;a href="mailto:

All users have a home project, yours is <%= link_to USER %>, in that project you can also create subprojects. New projects outside of your home project can get requested via &lt;a href="mailto:

**Comment by hennevogel** [comment]
@adrianschroeter please give your opinion: this PR is introducing home directories by default instead of forcing users to create one during the Project#new workflow

**Comment by adrianschroeter** [comment]
hm, there are setups out there where OBS is used for CI where this is not the case. Maybe it is not a problem for them, but what happens when the home projects get removed? Do they always get re-created on login? That will cause problems esp. for user accounts with a specific purpose only (eg. just handling some reviews or submitting code in official repos).

**Comment by hennevogel** [comment]
No, they are only automatically created after User creation. If you delete them, they are gone.

**Comment by adrianschroeter** [comment]
currently they are recreated when the user data gets updated, eg status or email adress change. If we do it only when the user gets created and when we also honour the configuration setting, if a user is allowed to create his home directory it would be fine for me.

See Configuration.allow_user_to_create_home_project  boolian

**Comment by hennevogel** [comment]
No they are not. They are only created after creation, not update

**Comment by hennevogel** [comment]
Okay @mdeniz so the callback should be after_create not after_save

**Comment by mdeniz** [comment]
Done

**Comment by hennevogel** [comment]
About the configuration setting. What does it mean exactly? It can't mean that there are no home projects because we are depending on home projects in tons of other places in the code.

**Comment by adrianschroeter** [comment]
> Does it mean 'normal' users can't do this but admins can?

Exactly. Or when the users are only supposed to work on defined projects or just have monitor/reviewer roles but should not be able to upload stuff.

**Comment by hennevogel** [comment]
Okay cool

**Pull request merged**